python/ray/serve/schema.py

```diff
@@ -186,6 +186,62 @@ def __eq__(self, other: Any) -> bool:
         return self._compute_hash() == other._compute_hash()
 
 
+@PublicAPI(stability="alpha")
+class TracingConfig(BaseModel):
+    """Tracing config for configuring serve tracing."""
+
+    class Config:
+        extra = Extra.forbid
+
+    export_path: str = Field(
+        default=None,
+        description=(
+            "An export path to export traces. Should be of the "
+            'form "module.submodule_1...submodule_n.'
+            'export_tracing". This is equivalent to '
+            '"from module.submodule_1...submodule_n import export_tracing.'
+        ),
+    )
+
+    enable_tracing: bool = Field(
+        default=False,
+        description=("Whether to enable tracing. Default to False."),
+    )
```

Comment:
Let's call this `enabled` (the `_tracing` suffix is redundant since this is nested inside of `TracingConfig`).